### PR TITLE
Implemented Mobilepay method

### PR DESCRIPTION
**Pull request by aait**
Implemented Mobilepay method

### CODE REVIEW

File: src/SwedbankPay/Core/Core.php [hunk 1]
```diff
@@ -13,13 +15,14 @@
 use SwedbankPay\Core\Library\Methods\Checkout;
 use SwedbankPay\Core\Library\Methods\Consumer;
 
-class Core implements CoreInterface
+class Core implements CoreInterface, MobilepayInterface
```
Comment (asbjornu) [inline src/SwedbankPay/Core/Core.php:18]: Why does `Core` implement the `MobilePayInterface` but none of the other payment instrument interfaces?

**Review by asbjornu — CHANGES_REQUESTED** [review]
The unit tests are failing. Why?

**Comment by asbjornu** [comment]
Also, doesn't this require SwedbankPay/swedbank-pay-sdk-php#15 to be completed and merged first?

File: src/SwedbankPay/Core/Core.php (hunk 1 above)
Comment (aait) [reply src/SwedbankPay/Core/Core.php:18]: @asbjornu Because that isn't ready yet, that in TODO list :-)

**Pull request merged**